vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -69,13 +92,63 @@ static int YogaLog(
 }
 #endif
 
+class LayoutConformanceVisitor : public ShadowNodeVisitor {
+  using Super = ShadowNodeVisitor;
+
+ public:
+  LayoutConformanceVisitor(YGErrata errata) : m_errata{errata} {}
+
+ protected:
+  void VisitCore(ShadowNodeBase *node) override {
+    if (!node->m_layoutConformance.has_value()) {
+      ApplyYogaConfig(node);
+      Super::VisitCore(node);
+    }
+  }
+
+ private:
+  void ApplyYogaConfig(ShadowNodeBase *node) {
+    const auto uiManager = EnsureNativeUIManager(node);
+    const auto yogaConfig = uiManager->GetYogaConfig(node->m_tag);
+    if (yogaConfig && YGConfigGetErrata(yogaConfig) != m_errata) {
+      YGConfigSetErrata(yogaConfig, m_errata);
+      uiManager->DirtyYogaNode(node->m_tag);
+      node->m_appliedLayoutConformance = true;
+    }
+  }
+
+  YGErrata m_errata;
+};
+
+class LayoutConformanceParentVisitor : public ShadowNodeVisitor {
+ public:
+  std::optional<bool> inheritedLayoutConformance = std::nullopt;
+
+ protected:
+  void VisitCore(ShadowNodeBase *node) override {
+    if (node->m_layoutConformance.has_value()) {
+      inheritedLayoutConformance = node->m_layoutConformance;
+    } else {
+      Visit(GetShadowNode(node->m_parent));
```

Comment:
Does this just traverse upward? Might make sense to just replace this with a loop, instead of overriding `ShadowNodeVisitor`, but with different behavior compared to superclass.

Reply:
Will clean this up later. Following patterns used by TextParentVisitor.